src/lib/browse/layers/areas/Pollution.svelte

```diff
@@ -4,10 +4,30 @@
   import { layerId } from "lib/maplibre";
   import { RasterLayer, RasterTileSource } from "svelte-maplibre";
   import OsOglLicense from "../OsOglLicense.svelte";
+  import { customUrlState } from "../url";
 
-  let show = false;
-  let opacity = 50;
-  let pollutant = "PM25_viridis";
+  type State = {
```

Comment:
Example of a more complex change. Any layer with more controls beyond show/hide has something like this. It's verbose, but I don't think it can really be much less code than this, without trying to encode as some very awkward JSON strings.